pkg/iptables/iptables.go

```diff
@@ -25,6 +25,7 @@ const (
 
 var (
 	supportsXlock       = false
+	iptablesPath        = ""
```

Comment:
good point.. I was thinking initialization to force type, but specifying type makes it cleaner, thx